packages/ui/src/components/radio-group-stacked.tsx

```diff
@@ -69,11 +69,12 @@ const RadioGroupStackedItem = React.forwardRef<
               'group-focus:outline-none',
               'group-focus-visible:ring-2 group-focus-visible:ring-ring group-focus-visible:ring-offset-2',
               'flex items-center justify-center',
+              'relative',
```

Comment:
Maybe it was already like this, but any reason why we separate base Tailwind classes into 3 different lines? We grouping layout, sizing, colours etc? I understand it for things like states, groups and pseudo classes, just not sure how granular we go? Either way, not blocking this :D